rcl_yaml_param_parser/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(rcl_yaml_param_parser)
+
+find_package(ament_cmake_ros REQUIRED)
+find_package(yaml REQUIRED)
+find_package(rcutils REQUIRED)
+
+include_directories(include ${yaml_INCLUDE_DIRS} ${rcutils_INCLUDE_DIRS})
+
+set(rcl_yaml_parser_sources
+  src/parser.c
+)
+
+add_library(
+  ${PROJECT_NAME}
+  ${rcl_yaml_parser_sources})
+ament_target_dependencies(${PROJECT_NAME} "yaml" "rcutils")
+
+install(TARGETS ${PROJECT_NAME}
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin)
+
+if(BUILD_TESTING)
+  if(NOT WIN32)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+  endif()
+
+  find_package(ament_cmake_gmock REQUIRED)
+  find_package(ament_cmake_gtest REQUIRED)
+  find_package(ament_cmake_pytest REQUIRED)
+  find_package(ament_lint_auto REQUIRED)
+  ament_lint_auto_find_test_dependencies()
+
+  set(extra_test_libraries)
+  set(extra_test_env)
+  set(extra_lib_dirs)
```

Comment:
Are any of these variables used ?

Reply:
No, lingering from rcutils CMake. Removed